test/devicecontroller/DefaultDeviceController.test.ts

```diff
@@ -2398,28 +2398,102 @@ describe('DefaultDeviceController', () => {
       element = videoElementFactory.create();
     });
 
-    it('connects the video stream to the video element', async () => {
+    it('connects the transform video stream to the video element if there is a transform device stream', async () => {
       const spy1 = sinon.spy(DefaultVideoTile, 'disconnectVideoStreamFromVideoElement');
       const spy2 = sinon.spy(DefaultVideoTile, 'connectVideoStreamToVideoElement');
       const gUM = sinon.spy(navigator.mediaDevices, 'getUserMedia');
+      domMockBehavior.createElementCaptureStream = new MediaStream();
+      domMockBehavior.mediaStreamTrackSettings.deviceId = stringDeviceId;
+      domMockBehavior.createElementCaptureStream.addTrack(new MediaStreamTrack());
+      // Connect the original video stream
       await deviceController.chooseVideoInputDevice(stringDeviceId);
       deviceController.startVideoPreviewForVideoInput(element);
-      expect(spy1.calledOnce).to.be.true;
+      const processor = new NoOpVideoFrameProcessor();
+      const device = new DefaultVideoTransformDevice(logger, stringDeviceId, [processor]);
+      // Connect the transform video stream
+      await deviceController.chooseVideoInputDevice(device);
+      deviceController.startVideoPreviewForVideoInput(element);
+
+      expect(device.getInnerDevice() === stringDeviceId);
+      expect(element.srcObject === (await gUM.returnValues.pop()));
+      expect(spy1.notCalled).to.be.true;
+      expect(spy2.calledTwice).to.be.true;
+      expect(gUM.calledOnce).to.be.true;
+      spy1.restore();
+      spy2.restore();
+      gUM.restore();
+    });
+
+    it('connects the active device video stream to the video element if there is no transform device stream', async () => {
```

Comment:
Can we just keep the original unit tests name `connects the video stream to the video element` and create a new one for transform device? Or say regular device or something.

Reply:
I already have one for transform device: `connects the transform video stream to the video element if there is a transform device stream`. 

I think I should rename the active device to original device. Active device is vague as both original and transform stream can be an active stream.